style.css

```diff
@@ -384,4 +384,84 @@ body {
     text-align: left;
     color: #344563;
   }
+
+  footer {
+    background-color: #6070ff;
+    padding-top: 103px;
+    border-radius: 100px 0 0 0;
+    background-image: url(./figmaImages/images/contactformshapes-mobile.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20px;
+  }
+
+  .form {
+    width: 87%;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+
+  .form h2 {
+    color: #fff;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .form p {
+    margin-top: 12px;
+    color: #ebebff;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  form {
+    width: 87%;
+    margin: auto;
+    padding-bottom: 45px;
+  }
+
+  .contact-form {
+    width: 100%;
+    padding: 45px 0 0 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contact-form input {
+    width: 100%;
+    margin-bottom: 20px;
+    padding: 15px 38px 15px 16px;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    color: #172b4d;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  .contact-form textarea {
+    width: 100%;
+    min-width: 100%;
+    max-width: 100%;
+    margin-bottom: 20px;
+    padding: 15px 38px 15px 16px;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    color: #b3bac5;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    min-height: 170px;
+  }
+
+  .myForm button {
+    margin: 0;
+    padding: 12px 16px;
+    width: auto;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] Good job so far 👏 . However, kindly remove the black boarder on the button to have a replica of the figma design 👍 
- [ ] I also noticed you didn't implemnet the hover, press and active features of the buttton to make it interactive just as said by the last reviwer. Kindly follow this [Link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), choose your template, and implement the interactions on the button 👍 

### Figma:
![image](https://github.com/gilberthappi/portfolio/assets/50990925/35b20f65-c6d6-4cc8-932d-731b7ad9ac4d)

### Your work:
![image](https://github.com/gilberthappi/portfolio/assets/50990925/31992006-04c7-4ac7-8247-85f04884c3ea)

### Interactions: This [Link](https://freefrontend.com/css-button-click-effects/) might help
![image](https://github.com/gilberthappi/portfolio/assets/50990925/8d47b5cc-de68-4a13-ad9e-fdf6a60ee7a3)

Reply:
okey
thanks